moped-editor/src/views/projects/projectView/ProjectView.js

```diff
@@ -402,7 +449,7 @@ const ProjectView = () => {
                 <div className={classes.root}>
                   <Box p={4} pb={2}>
                     <Grid container>
-                      <Grid item xs={12}>
+                      <Grid item xs={11}>
```

Comment:
I'm curious why this change was needed.

Reply:
It was not, thank you for catching that!

Reply:
Actually removing that caused formatting issues. I set that grid item to 11 so I could create another grid item for the following button and set that to 1. I'm returning it to that state but please let me know if I'm missing something @mateoclarke